pkg/alpine/mirrors/3.16/main

```diff
@@ -202,3 +202,10 @@ zlib-static
 zstd-dev
 zstd-libs
 zstd-static
+openrc
+open-iscsi
+pm-utils
+netcat-openbsd
+gnutls-utils
+boost1.78-dev
+parted 
```

Comment:
Please check the yetus annotations in https://github.com/lf-edge/eve/pull/3264/files. There are blanks complaints on multiple files.

Reply:
Still have yetus issue here